charts/dependency-track/README.md

```diff
@@ -1,13 +1,39 @@
 # Dependency-track
 
 ## Installing
-
+To install this chart on Kubernetes, use the following commands:
 ```
 kubectl create namespace dependency-track
-helm install dependency-track evryfs-oss/dependency-track --namespace dependency-track
+helm install dependency-track evryfs-oss/dependency-track [--set ingress.enabled=true --set ingress.host=<desired_URL>] --namespace dependency-track
 ```
 
-## Configuration
+**Note:** Depending on how you install the chart, the `API_BASE_URL` value in values.yaml may need to be amended.
+ - If `ingress.enabled=true` then it can be left as `API_BASE_URL=""`
+ - If ingress is not used, or separate ingress URLs are desired for the frontend and backend, then the `API_BASE_URL` will need to be set as the URL for the Backend
+ - If you wish to use port-forwarding then `API_BASE_URL=http://127.0.0.1:<port>`, see values.yaml for more information
+
+To install this chart on OpenShift, using the `anyuid` or `nonroot` SCCs, use the following commands (choosing anyuid or nonroot as the SCC):
+```
+oc adm policy add-scc-to-user <anyuid|nonroot> -z dependency-track-frontend
```

Comment:
I believe this mutates an existing resource? Better to use rolebinding towards the clusterroles?

Reply:
Would you like to specify this is used instead:
```
oc create role deptrack --verb=use --resource=scc --resource-name=<anyuid|nonroot> -n dependency-track
oc create rolebinding admin --role=deptrack --serviceaccount=dependency-track-frontend --serviceaccount=dependency-track-apiserver --serviceaccount=dependency-track-postgresql
```

Reply:
It can be like this:
```
k create rolebinding anyuid --clusterrole=system:openshift:scc:anyuid --group=system:serviceaccounts:<RELEASENAMESPACE>
```

Reply:
Ok, cool. I've had a play around and I think this one is safer as more specific - otherwise we risk over-assigning to whole namespace with the other:
```
oc create rolebinding anyuid --clusterrole=system:openshift:scc:anyuid --serviceaccount=<namespace>:dependency-track-frontend --serviceaccount=<namespace>:dependency-track-apiserver --serviceaccount=<namespace>:dependency-track-postgresql
```